packages/legacy-client/src/ontology-runtime/aggregations/ComputeStep.ts

```diff
@@ -14,14 +14,10 @@
  * limitations under the License.
  */
 
-import type { Auth } from "../../oauth-client";
-import type { BaseObjectType, ObjectSetDefinition } from "../baseTypes";
-import {
-  type AggregateObjectsError,
-  type OntologyMetadata,
-  OntologyProvider,
-  type Result,
-} from "../ontologyProvider";
+import type { ObjectSetDefinition } from "../baseTypes";
+import { type AggregateObjectsError, type Result } from "../ontologyProvider";
+import { aggregate } from "../ontologyProvider/calls/aggregate";
+import type { ClientContext } from "../ontologyProvider/calls/ClientContext";
```

Comment:
Not needed now but we should probably slowly reorganize the old code out of the `ontology-runtime` subfolder as we pull things in from it